test/libsolidity/syntaxTests/immutable/getter.sol

```diff
@@ -1,5 +1,5 @@
 contract C {
-    uint immutable public x;
+    uint immutable public x = 1;
 }
 // ----
 // UnimplementedFeatureError: NONE
```

Comment:
Please move this test out of this PR.

Reply:
The test already exists in the repo, I am just fixing it so it doesn't fail. Should I remove it completely?

Reply:
Yes, please move it to its own PR. We should not have tests that are expected to be failing.